.github/workflows/python.yml

```diff
@@ -36,20 +39,36 @@ jobs:
         key: ${{ runner.os }}-pip-${{ matrix.python-version }}-${{ hashFiles('Pipfile.lock') }}
       if: matrix.python-version == 'pypy3'
     - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install .[dev]
-    - name: Lint with flake8
-      run: |
-        # Use settings from mesas .flake8 file
-        flake8 . --count --show-source --statistics
-    - name: Lint with black
-      if: matrix.python-version == '3.8'
-      run: |
-        pip install black
-        black --check --exclude=mesa/cookiecutter-mesa/* .
+      run: pip install .[dev]
     - name: Test with pytest
-      run: |
-        pytest --cov=mesa tests/ --cov-report=xml
+      run: pytest --cov=mesa tests/ --cov-report=xml
     - name: Codecov
       uses: codecov/codecov-action@v1.0.15
+
+  lint-flake:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.10"
+    - name: Install dependencies
+      run: pip install .[dev]
```

Comment:
Same question about the necessity as below.

Reply:
This line installs it. The one below runs it. So I think this is necessary.